### PR TITLE
WIP: BlockRegistry restructure to allow for easier texture management and loading in of blocks via lua.

### PR DESCRIPTION
**Pull request by beeperdeeper089**

### CODE REVIEW

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 15
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by beeperdeeper089** [comment]
codacy can suck it

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

**Comment by apachano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GentenStudios/Phoenix/pullRequest?prid=4765477&bid=15886136)

File: Engine/Include/Quartz2/Voxels/ChunkManager.hpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Genten Studios
```
Comment (tobyplowy) [inline Engine/Include/Quartz2/Voxels/ChunkManager.hpp]: Hehem.. 2020 ;D

Comment (beeperdeeper089) [reply Engine/Include/Quartz2/Voxels/ChunkManager.hpp]: will be fixed later when i actually pr this stuff in

Comment (tobyplowy) [reply Engine/Include/Quartz2/Voxels/ChunkManager.hpp]: Ok.

**Pull request closed without merging**